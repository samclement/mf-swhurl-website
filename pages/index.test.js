/* eslint-env jest */

import React from 'react'
import { render } from 'enzyme'
import 'jest-styled-components'

import Index from './index.js'

describe('With Snapshot Testing', () => {
  it('Index page shows "swhurl.com" H1', () => {
    const wrapper = render(<Index />)
    expect(wrapper).toMatchSnapshot()
  })
})
