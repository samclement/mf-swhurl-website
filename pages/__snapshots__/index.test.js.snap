// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing Index page shows "swhurl.com" H1 1`] = `
.c1 {
  color: deeppink;
}

.c0 {
  color: 'deeppink';
}

<div>
  <nav>
    <a
      class="c0"
      href="/"
    >
      Home
    </a>
     | 
    <a
      class="c0"
      href="/about"
    >
      About
    </a>
     | 
    <a
      class="c0"
      href="/contact"
    >
      Contact
    </a>
     | 
    <a
      class="c0"
      href="/login"
    >
      Login
    </a>
  </nav>
  <h1
    class="c1"
  >
    swhurl.com
  </h1>
  <p>
    Example next.js app, including:
  </p>
  <ul>
    <li>
      styled components
    </li>
    <li>
      jest snapshot testing
    </li>
  </ul>
</div>
`;
